packages/backend/src/flags/flags.controller.ts

```diff
@@ -86,9 +86,7 @@ export class FlagsController {
     @Param('envId') envId: string,
     @Param('flagId') flagId: string,
   ): Promise<any> {
-    const rawHits = await this.flagService.listFlagHits(envId, flagId);
-
-    return rawHits;
+    return await this.flagService.listFlagHits(envId, flagId);
```

Comment:
On this one, it can be a solution to remove `async/await` since `listFlagHits` is already returning a Promise.
If you keep it, it's consistency accros methods which is also nice :)

```suggestion
    return this.flagService.listFlagHits(envId, flagId);
```

![Capture d’écran 2022-08-14 à 18 36 52](https://user-images.githubusercontent.com/22884251/184546365-924087e9-7d62-453c-b863-4d0f5a5b6ba9.png)

At least, the up `Promise<any>` can be removed in order to let type inference do the job.
Which mean explicit type declaration in `packages/frontend/app/routes/dashboard/projects/$id/environments/$env/flags/$flagId/insights.tsx:80` can also be removed.